Client/Application/Preferences.swift

```diff
@@ -45,6 +45,12 @@ extension Preferences {
         /// Defaults to nil, meaning the user has not been given the choice yet
         static let duckDuckGoPrivateSearch = Option<Bool?>(key: "popups.ddg-private-search", default: nil)
     }
+    final class AppState {
+        /// A flag for determining if the app exited during user interaction in the previous session
+        ///
+        /// Value should only be checked on launch
+        static let crashTrackingFlag = Option<Bool>(key: "appstate.crash-tracking-flag", default: false)
```

Comment:
This naming is confusing IMO. Not quite sure what `true` means. Maybe `didCrashFlag`/`didCrash`? or `appCleanExit` (reverse logic)?